packages/terra-badge/tests/specs/badge.js

```diff
@@ -0,0 +1,39 @@
+/* global browser, describe, it, expect, before, Terra */
+const viewports = Terra.viewports('tiny', 'medium');
+
+const shouldBeAccessible = () => {
+  it('is accessible', () => expect(browser.axe()).to.be.accessible());
+};
+
+const shouldBeVisuallyConsistent = () => {
+  it('[0] checks visual comparison', () => {
+    const screenshots = browser.checkElement('[data-reactroot]', { viewports });
```

Comment:
Moving this to toolkit, I'd like to see this updated to pass in the desired checkElement with the fallback to '[data-reactroot]'